scripts/scil_mti_maps_MT.py

```diff
@@ -254,12 +254,20 @@ def main():
     if args.in_acq_parameters:
         flip_angles = np.asarray(args.in_acq_parameters[:2]) * np.pi / 180.
         rep_times = np.asarray(args.in_acq_parameters[2:]) * 1000
+        if rep_times[0] > 10000 or rep_times[1] > 10000:
+                logging.warning('Given repetition times do not seem to '
+                                'be given in seconds. MTsat results might be '
+                                'affected.')
     elif args.in_jsons:
         rep_times = []
         flip_angles = []
         for curr_json in args.in_jsons:
             acq_parameter = get_acq_parameters(curr_json,
                                                ['RepetitionTime', 'FlipAngle'])
+            if acq_parameter[0] > 10:
+                logging.warning('Repetition time found in {} does not seem to '
+                                'be given in seconds. MTsat results might be '
+                                'affected.'.format(curr_json))
```

Comment:
With INFO:root messages, the Warning message goes unnoticed, perhaps with an \n before and/or after it?